src/shared/utils.ts

```diff
@@ -105,6 +105,11 @@ export type ThemeColor =
   | "gray"
   | "gray-dark";
 
+export interface ErrorPageData {
+  error?: string;
+  adminMatrixIds?: string[];
```

Comment:
What's this one for?

Reply:
![500-error-matrix](https://github.com/LemmyNet/lemmy-ui/assets/28871516/b14f60ef-8ecb-4898-a5dd-ce529bf61cad)
If there's a server error but the site is fetched successfully and there are admins with matrix IDs, the error page will list those matrix IDs for users to contact.

Reply:
Gotcha, cool.